packages/mask/src/plugins/EVM/UI/Web3State/getAssetsFn.ts

```diff
@@ -38,8 +37,20 @@ export const getFungibleAssetsFn =
         const web3 = new Web3(provider)
         const { BALANCE_CHECKER_ADDRESS } = getEthereumConstants(chainId)
         const { NATIVE_TOKEN_ADDRESS } = getTokenConstants()
-        const dataFromProvider = await context.getAssetsList(address, FungibleAssetProvider.DEBANK)
-        const assetsFromProvider: Web3Plugin.Asset<Web3Plugin.FungibleToken>[] = dataFromProvider.map((x) => ({
+        const socketId = `mask.fetchFungibleTokenAsset_${address}`
+        let dataFromProvider = await socket.sendAsync<Web3Plugin.Asset<Web3Plugin.FungibleToken>>({
+            id: socketId,
+            method: 'mask.fetchFungibleTokenAsset',
+            params: {
+                address: address,
+                pageSize: 10000,
+            },
+        })
+        if (!dataFromProvider.length) {
+            // @ts-ignore getAssetList Asset[]
+            dataFromProvider = await context.getAssetsList(address, FungibleAssetProvider.DEBANK)
+        }
+        const assetsFromProvider: Web3Plugin.Asset<Web3Plugin.FungibleToken>[] = dataFromProvider.map((x: any) => ({
```

Comment:
Avoid `any` if possible.

Reply:
The context.getAssetsList return type is `Asset FungibleTokenDetailed`, It is different with `await socket.sendAsync<Web3Plugin.Asset<Web3Plugin.FungibleToken>>()`. This is workaround.